types/example-app/CustomLink.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 interface Props {
   href: string;
-  active: boolean;
+  active?: boolean;
```

Comment:
`active` should always be there

Reply:
it's not in the types, and it can't. the reason being that `activeFieldName` is customizable

Reply:
erk, do we need two of these, then?

when rendered as a function child, do we get the correct type of `active: boolean`?

Reply:
yes